reference/catalog/products_catalog.v3.yml

```diff
@@ -0,0 +1,8866 @@
+                    count: 2
+                    per_page: 50
+                    current_page: 1
+                    total_pages: 1
+                    links:
+                      current: '?page=1&limit=50'
+        '404':
+          description: |
+            The resource was not found.
+          content:
+            application/json:
+              schema:
+                title: Not Found
+                type: object
+                properties:
+                  status:
+                    type: integer
+                    description: |
+                      404 HTTP status code.
+                  title:
+                    type: string
+                    description: The error title describing the particular error.
+                  type:
+                    type: string
+                  instance:
+                    type: string
+                description: Error payload for the BigCommerce API.
+    post:
+      tags:
+        - Metafields
+      summary: Create a Product Metafield
+      description: |-
+        Creates a *Product Metafield*.
+
+        **Required Fields:**
+        * permission_set
+        * namespace
+        * key
+        * value
+
+        **Read-Only Fields**
+        * id
+
+        **Note:** The maxiumum number of metafields allowed on each order, product, category, variant, or brand is 250 per client ID. For more information, see [Platform Limits](https://support.bigcommerce.com/s/article/Platform-Limits) in the Help Center.
+      operationId: createProductMetafield
+      parameters:
+        - $ref: '#/components/parameters/ContentType'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/metafield_Base'
+        required: true
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                title: Metafield Response
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/metafield_Full'
+                  meta:
+                    $ref: '#/components/schemas/metaEmpty_Full'
+              example:
+                data:
+                  id: 8
+                  key: location_id
+                  value: 'Shelf 3, Bin 5'
+                  namespace: Inventory Namespace
+                  permission_set: read
+                  resource_type: product
+                  resource_id: 158
+                  description: Where products are located
+                  date_created: '2018-09-13T16:42:37+00:00'
+                  date_modified: '2018-09-13T16:42:37+00:00'
+                meta: {}
+        '409':
+          description: |
+            The `Metafield` was in conflict with another `Metafield`. This can be the result of duplicate unique key combinations of the app's client id, namespace, key, resource_type, and resource_id.
```

Comment:
```suggestion
            The `Metafield` was in conflict with another `Metafield`. This can be the result of duplicate unique key combinations of the app's client ID, namespace, key, resource_type, and resource_id.
```